payments-core/src/main/java/com/stripe/android/model/DeferredIntentParams.kt

```diff
@@ -41,13 +41,6 @@ data class DeferredIntentParams(
         }
     }
 
-    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-    enum class CaptureMethod(val code: String) {
-        Automatic("automatic"),
-        AutomaticAsync("automatic_async"),
-        Manual("manual"),
-    }
```

Comment:
Decided to use `PaymentIntent.CaptureMethod` in this internal class instead of another new type. It makes the comparison in the validator easier.